misc-scripts/xref_mapping/XrefParser/ZFINDescParser.pm

```diff
@@ -22,58 +22,73 @@ package XrefParser::ZFINDescParser;
 use strict;
 use warnings;
 use Carp;
-use POSIX qw(strftime);
-use File::Basename;
-use File::Spec::Functions;
+use Text::CSV;
 
-use base qw( XrefParser::BaseParser );
+use parent qw( XrefParser::BaseParser );
 
 
 sub run {
-
   my ($self, $ref_arg) = @_;
+
   my $source_id    = $ref_arg->{source_id};
   my $species_id   = $ref_arg->{species_id};
   my $files        = $ref_arg->{files};
-  my $verbose      = $ref_arg->{verbose};
+  my $verbose      = $ref_arg->{verbose} // 0;
 
-  if((!defined $source_id) or (!defined $species_id) or (!defined $files) ){
-    croak "Need to pass source_id, species_id and files as pairs";
+  if ( (!defined $source_id) or (!defined $species_id) or (!defined $files) ) {
+    croak 'Need to pass source_id, species_id and files';
```

Comment:
why was the double quote replaced by a single quote?
double quotes are usually safer and it is used that way further down the code (line 56 for example)

Reply:
no good reason really, it's only because there is no variable interpolation happening and I got the habit of double quoting only when I need to...
will change it back.